ui/src/design-system/components/dialog/dialog.module.scss

```diff
@@ -16,7 +16,7 @@
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  max-height: calc(100vh - 128px);
+  height: calc(100vh - 128px);
```

Comment:
Not sure if it's documented elsewhere, but maybe move that 128px to a variable and name it something understandable. Where does that number 128 come from, and why is it just that?

Reply:
Will do! It's the dialog padding * 2 :)

<img width="1387" alt="Screenshot 2023-05-30 at 08 31 37" src="https://github.com/RolnickLab/ami-platform/assets/11680517/7b038e3d-0b1b-4234-b6c9-4a9541013cdf">